common/artifacts/901_wc_artifacts_weapon_onehanded.txt

```diff
@@ -98,4 +124,63 @@ serathil = {
 
 	picture = "GFX_serathil"
 	slot = weapon
-}
\ No newline at end of file
+}
+the_bladefist = {
+	quality = 3
+	flags = { combat_weapon sword }
+	
+	stacking = no
+	
+	active = {
+		trait = one_handed
+		shared_physical_weapon_trigger = yes
+	}
+	
+	martial = 1
+	combat_rating = 16
+	
+	monthly_character_prestige = 0.15 
+
+	command_modifier = {
+		morale_offence = 0.06
+	}
+
+	picture = "GFX_the_bladefist"
+	
+	slot = weapon 
+
+}
```

Comment:
Why deviate that much from the prosthetic blade hand we had before?

Reply:
Why not?

Reply:
Because it's basically the same thing/intention?